### PR TITLE
Add Hungarian translation

### PR DESCRIPTION
**Pull request by Kokan**

### CODE REVIEW

File: po/hu.po [hunk 1]
```diff
@@ -0,0 +1,214 @@
+# Hungarian translation
+# Copyright (C) 2022 Kokan <kokaipeter@gmail.com>
```
Comment (dusansimic) [inline po/hu.po:2]: This is the copyright for the project, not for the translation author :). Leave it as it was.

**Review by dusansimic — CHANGES_REQUESTED** [review]

File: po/hu.po (hunk 1 above)
Comment (Kokan) [reply po/hu.po:2]: Sure.

**Review by dusansimic — APPROVED** [review]

**Comment by dusansimic** [comment]
Thanks! Great work 😁

**Comment by Kokan** [comment]
Thanks, feel free to ping me if update is needed.

**Pull request merged**